Remotion/ObjectBinding/Web.IntegrationTests/BocEnumValue/AccessibilityBocEnumValueTest.cs

```diff
@@ -0,0 +1,291 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+// 
+// The re-motion Core Framework is free software; you can redistribute it 
+// and/or modify it under the terms of the GNU Lesser General Public License 
+// as published by the Free Software Foundation; either version 2.1 of the 
+// License, or (at your option) any later version.
+// 
+// re-motion is distributed in the hope that it will be useful, 
+// but WITHOUT ANY WARRANTY; without even the implied warranty of 
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
+// GNU Lesser General Public License for more details.
+// 
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+// 
+using System;
+using NUnit.Framework;
+using Remotion.ObjectBinding.Web.Development.WebTesting.IntegrationTests;
+using Remotion.Web.Development.WebTesting.Accessibility;
+using Remotion.Web.Development.WebTesting.ExecutionEngine.PageObjects;
+using Remotion.Web.Development.WebTesting.FluentControlSelection;
+
+namespace Remotion.ObjectBinding.Web.IntegrationTests.BocEnumValue
+{
+  [TestFixture]
+  public class AccessibilityBocEnumValueTest : IntegrationTest
```

Comment:
Validate-Test got dropped. Why. Actually, do we have Accessibility Tests for "a validation error occured" with the other control tests? if not, we should add them to ensure the markup that includes the business-logic-validation errors is valid.

Reply:
The test named "Validate" was dropped because it was just a copy-pasta of another test without even calling `Validate`, it was an oversight.


I went over all the Boc tests again, the following controls are missing validator tests:

* AutocompleteReferenceValue
* BooleanValue
* Checkbox
	* Is it even possible to trigger a validation error with this control?
* DateTimeValue
* List
* ListAsGrid
* ReferenceValue
* TreeView
	* Is it even possible to trigger a validation error with this control?

Meaning the following controls do have a validator test:

* EnumValue
* TextValue
* MultiLineTextValue

I will add the missing validator tests now

Reply:
TreeView is readOnly, so, no.
BocCheckbox: you could probably build something with a custom validator. Probably easier to show than tell.